docs/datatypes/validated.md

```diff
@@ -589,8 +589,61 @@ validatedMonad.tuple2(Validated.invalidNec[String, Int]("oops"), Validated.inval
 ```
 
 This one short circuits! Therefore, if we were to define a `Monad` (or `FlatMap`) instance for `Validated` we would
-have to override `ap` to get the behavior we want. But then the behavior of `flatMap` would be inconsistent with
-that of `ap`, not good. Therefore, `Validated` has only an `Applicative` instance.
+have to override `ap` to get the behavior we want. 
+
+```scala mdoc:silent:nest
+import cats.Monad
+
+implicit def inconsistentValidatedMonad[E: Semigroup]: Monad[Validated[E, *]] =
```

Comment:
This name spoils the ending 😛 maybe we can call it `accumulatingValidatedMonad` or something?